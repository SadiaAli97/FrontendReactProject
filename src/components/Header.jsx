import * as React from 'react';
import Box from '@mui/material/Box';
import Grid from '@mui/material/Grid';
import '../css/header.css';

export function Header({heading, subheading, count}) {

  return (
    <Box sx={{ flexGrow: 1 }}>
      <Grid container spacing={2}>
        <Grid item xs={8}>
            <div className='heading'>
              {heading}
            </div>
            <div className='subheading'>
              {subheading}
            </div>  
        </Grid>
        <Grid item xs={4}>
            <div className='listcount'>
              {count}
            </div>
        </Grid>
      </Grid>
    </Box>
  );
}
