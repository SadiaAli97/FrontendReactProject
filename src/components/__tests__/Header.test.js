import TestRenderer from 'react-test-renderer';
import { Header } from '../Header';

it('renders Header ', () => {
	const testRenderer = TestRenderer.create(
		<Header
		  heading='testheading'
		  subheading='testsubheading'
		  count='7'
		/>
	);
  expect(testRenderer.toJSON()).toMatchSnapshot()
});

