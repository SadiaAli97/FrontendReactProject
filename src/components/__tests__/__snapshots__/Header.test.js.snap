// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header  1`] = `
<div
  className="MuiBox-root css-i9gxme"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8 css-vok43g-MuiGrid-root"
    >
      <div
        className="heading"
      >
        testheading
      </div>
      <div
        className="subheading"
      >
        testsubheading
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-gj1fbr-MuiGrid-root"
    >
      <div
        className="listcount"
      >
        7
      </div>
    </div>
  </div>
</div>
`;
